src/devices/philips_AJ7010.c

```diff
@@ -0,0 +1,164 @@
+/* 
+ * Philips outdoor temperature sensor -- used with various Philips clock 
+ * radios (tested on AJ7010).
+ * This is inspired from the other Philips driver made by Chris Coffey.
+ *
+ * A complete message is 43 bits:
+ *      4-bit initial preamble, always 0
+ *      4-bit packet separator, always 0
+ *      Packets are repeated 3 times.
+ * 
+ * 43-bit data packet format:
+ *
+ * 00001000 10100100 01000101 00101001 110 : g_philips_21.1_ch2_B.cu8
+ * 00001011 01001111 10000101 00100001 111 : g_philips_21.4_ch1_C.cu8
+ * 00001011 01000000 10100100 11001111 001 : gph_bootCh1_17.cu8
+ * 00001000 10100011 11000100 11001111 101 : gph_bootCh2_17.cu8
+ * 00000110 11011100 01100100 10111110 000 : gph_bootCh3_17.cu8
+ *
+ * Data format is:
+ * 0000cccc ccc????? ???????ttt tttttttt ttt
+ *
+ * c - channel: 0x5A=channel 1, 0x45=channel 2, 0x36=channel 3 (7 bits)
+ * t - temperature in ADC value that is then converted to deg. C. (14 bits)
+ * s - CRC: non-standard CRC-?, poly 0x?, init 0x?
+ *  
+ * Copyright (C) 2018 Nicolas Jourden <nicolas.jourden@laposte.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
```

Comment:
The rtl_433 project is licensed under the GPL-2.0 only license. Are those licenses compatible?